r/neotoma_tutorial.Rmd

```diff
@@ -396,32 +489,24 @@ extinct_tol <- extinct_tol[!is.na(extinct_tol)]
 
 ### Make data frame to merge with 
 
-[TODO] THis is an old way of making a dataframe.  Is there a more efficient way? This is super unreadable and confusing code. Maybe skip this part and upload the dataframe after?
+[TODO] Switch from read/write to using this if you think this approach is ok.
```

Comment:
If you think this version is okay, we can make eval = T and avoid reading in an intermediate file.